packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 ## Unreleased
 
+* Added
+  * Added back `width: 100%` to flowsheet result cell styles removed in 1.17.0.
+
+* Changed
+  * Updated FlowsheetResultCell useLayoutEffect to useEffect.
+  * Updated FlowsheetResultCell  dependencies and component states from useEffect.
```

Comment:
I don't think these changes are relevant or informative to consumers. We can probably change this to:
```
* Fixed:
  * Fixed infinite re-render bug.
```

Reply:
Updated [3ac99ac](https://github.com/cerner/terra-clinical/pull/902/commits/3ac99ac16387f26faca19146515324242f213fc4) and [31b3a3c](https://github.com/cerner/terra-clinical/pull/902/commits/31b3a3c0c7c3e8f24b5b3577489f8ea553bd3e1e)